core/quality/morqa_transport.go

```diff
@@ -120,7 +139,7 @@ func sessionTokensToMetricsEvent(ctx sessionTokensContext) (string, *metrics.Eve
 		IsProvider: false,
 		Metric: &metrics.Event_SessionTokensPayload{
 			SessionTokensPayload: &metrics.SessionTokensPayload{
-				Tokens: ctx.Tokens.Text(10),
+				Tokens: ctx.Tokens.Uint64(),
```

Comment:
What is `Tokens` in this context? If it is related to MYST tokens, then `Uint64` is not enough.

Reply:
This is sort of weird, I used to be a string, but after generating the new metrics, it turned in to an uint64, I didn't really edit it 🤷 
https://github.com/mysteriumnetwork/metrics/commit/544a1e13d6651b963d257aafb419282c84f434da#diff-6039342a2ba47b726e1134b8ead7c00196fd9cefd9dd925b98e3cf01f21b4e2cR43